src/constants/main.ts

```diff
@@ -14,13 +14,33 @@ const FORM_TABLE_FIELDS = [
         value: 'email',
     },
 ];
+const USER_TABLE_TITLE = [
+    ...FORM_TABLE_FIELDS,
+    { label: 'Role', value: 'role' },
+];
+const COINS_TABLE_TITLE = [
+    {
+        label: 'Name',
```

Comment:
not aa good idea to keep objects of different structure inside the structure. add value for this and the following item even if you don't use them anywhere in the code

Reply:
Added an interface, as well as a toSort parameter.